src/encode_ataqc.py

```diff
@@ -200,11 +203,11 @@ def ataqc():
         flagstat_map = parse_flagstat_qc(args.flagstat_log)
         with open(args.flagstat_log,'r') as fp:
             flagstat = fp.read()
-        mapped_count = flagstat_map['mapped']
+        #mapped_count = flagstat_map['mapped']
```

Comment:
It's usually best not to leave commented-out lines of code.  Here you're assigning mapped_count at a new place (above).  There's no need to leave this comment here to indicate that this is where it was assigned before.  The git history is adequate for that.

Reply:
sounds good, have removed in latest commit!